pinot-common/src/test/java/com/linkedin/pinot/common/utils/TarGzCompressionUtilsTest.java

```diff
@@ -122,4 +128,61 @@ public void testEmptyDirectory()
     Assert.assertEquals(segmentFiles.length, 0);
 
   }
+
+  @Test
+  public void testBadFilePath() throws Exception {
```

Comment:
are we cleaning the files created in tearDown

Reply:
We have a `teardownTest` method that deletes TEST_DIR, which contains all the test files. It will be called when each unit test is finished.